### DISCUSSION TITLE
project 3

### DISCUSSION THREAD

**Discussion by mikatakahashi** [IBAHCM/RPiR#311] [Archive] [answered]
Hi @soniamitchell @richardreeve,

Just a very quick question about project 3. 
On Tasks where you explain what to put in R as an equation for Shannon entropy, it says exp(sum(p * log(p))), but I was wondering if it should be exp(- sum(p * log(p)))? Let me know if I'm mistaken.

**Comment by soniamitchell** [comment, marked as answer]
Hey @mikatakahashi, you mean this bit?

>which is to say exp(sum(p * log(p))) in R, where p is the vector of non-zero species proportions.

Yes! That's a typo. The equation above is right, this bit is missing the minus sign. I'll fix it now, thanks 👍

**Reply by richardreeve** [reply]
Thanks for spotting that @mikatakahashi!
